extension/src/frontend/index.ts

```diff
@@ -169,39 +199,27 @@ function handleInputChange(target: HTMLInputElement) {
   })
 }
 
-window.addEventListener(
-  'change',
-  (ev) => {
-    if (shouldSkipEvent(ev)) {
-      return
-    }
-
-    if (ev.target instanceof HTMLTextAreaElement) {
-      handleTextAreaChange(ev.target)
+manager.capture('change', (ev) => {
```

Comment:
Removing that third `{ capture: true }` argument caused a lot of reformatting here. In reality, only the `shouldSkipEvent` was removed because that's handled by the `WindowEventManager`.